.github/workflows/sync-config-schema.yaml

```diff
@@ -53,13 +67,14 @@ jobs:
           GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
 
       - name: Set up Go
-        if: ${{ steps.release.outputs.update_main == 'false' }}
+        if: ${{ steps.release.outputs.is_alpha_version == 'false' }}
```

Comment:
`steps.release.outputs.is_stable_version == 'false' && && steps.release.outputs.is_alpha_version == 'false' `?

Reply:
no, we want to do this on betas, RC and full versions (see my comment below)